core/storage/user/gae_models_test.py

```diff
@@ -173,6 +195,11 @@ def test_instance_stores_correct_data(self):
         self.assertEqual(
             query_model.edited_fewer_than_n_exps, edited_fewer_than_n_exps)
 
+        query_models, _, _ = user_models.UserQueryModel.fetch_page(
+            1, None)
+        self.assertEqual(len(query_models), 1)
+        self.assertEqual(query_models[0].submitter_id, 'submitter')
```

Comment:
I am seeing this pattern in general where there is a single line of trivial validation for a model that is returned by a GET method.

This should not be the case. Validate as much as possible, in general, instead. There should be justification for anything that is not validated.

Also, I'm not sure why these statements are added to the existing test. If the point is to test fetch_page instead, then please create a new test method for this, and actually try to test fetch_page robustly. This includes using multiple pages and trying various values for the arguments. Remember that the coverage score is only a proxy; the point is to get well-tested code that prevents regressions in the future.

Reply:
Done